resources/sass/abstracts/_variables.scss

```diff
@@ -26,5 +26,21 @@ $color-white: rgb(255, 255, 255);
 $color-white-gray: rgb(235, 235, 236);
 $color-black-gray: rgb(77, 77, 79);
 $color-black: rgb(35, 31, 32);
+$color-black-light: rgb(45, 38, 40);
+$color-black-true: rgb(0, 0, 0);
 
 $transition-duration: 0.3s;
+
+$page-margins: 1600px;
+
+$breakpoint-tablet: 768px;
+$breakpoint-mobile: 480px;
+
+$font-size-xxl: 7rem;
+$font-size-xl: 5rem;
+$font-size-lg: 2.5rem;
+$font-size-md: 1.4rem;
```

Comment:
Šis varētu būt 1 rem

Reply:
Jūs selectojāt 4 font sizes, nezinu kuru likt uz 1rem. Bet samazināju overall sizes uz šiem:
6rem, 4.5rem, 2.2rem, 1.2rem, 0.8rem